src/EvtGenNtuplizer_GenInfo.cc

```diff
@@ -170,13 +181,6 @@ EvtGenNtuplizer::FillGen( const edm::Event& iEvent, const edm::EventSetup& iSetu
     // ----- b' decay mode  --------------------------------------------------
     // b' decay - 0:other, 1:tW, 2:cW, 3:bZ 4:bH
     if( pdgId == +7 ){
-      EvtInfo.McbprimeMass[0] = it_gen->p4().mag();
-      // b' decay mode      - 0: others, 1: tW, 2: cW, 3: bZ, 4: bH
-      if( dauId1 == 6 && dauId2 == 24 ){ EvtInfo.McbprimeMode[0] = 1; }
-      if( dauId1 == 4 && dauId2 == 24 ){ EvtInfo.McbprimeMode[0] = 2; }
-      if( dauId1 == 5 && dauId2 == 23 ){ EvtInfo.McbprimeMode[0] = 3; }
-      if( dauId1 == 5 && dauId2 == 25 ){ EvtInfo.McbprimeMode[0] = 4; }
-      if( EvtInfo.McbprimeMode[0] == 0 ){ continue; }
       if( dauId1 == 6 ){
         if( abs( dau1->daughter( 0 )->pdgId() ) == 5 ){
           MCDaughters[0] = dau1->daughter( 0 );
```

Comment:
Check if MCDaughter[X] is ever used to fill in the ntuple variables. Remove the empty logic statement otherwise.